client/src/pages/cejst.tsx

```diff
@@ -15,45 +15,51 @@ const CEJSTPage = ({location}: IMapPageProps) => {
-              <p>
-                Just Progress helps identify and prioritize communities across the United States and U.S. territories
-                that have been historically overburdened and underserved. These communities will receive 40% of
-                the benefits from investments in key areas outlined by the
-                &nbsp;
-                <a
-                  href={'https://www.whitehouse.gov/briefing-room/' +
-                    'presidential-actions/2021/01/27/' +
-                    'executive-order-on-tackling-the-climate-' +
-                    'crisis-at-home-and-abroad/'}
-                  target={'_blank'}
-                  rel={'noreferrer'}>
-                  Executive Order on Tackling the Climate Crisis at Home and Abroad
-                </a>.
-              </p>
-              <p>
-                Download the Just Progress packet or explore the map below to see the list of prioritized communites. To
-                learn more about how these communities were prioritized check out the
-                &nbsp;
-                <a
-                  href={'./methodology'}>
-                  Methodology
-                </a>
-                &nbsp;
-                page.
-              </p>
-            </div>
-            <DownloadPacket />
+      <section>
+        <h2>Just Progress communities</h2>
+        <div className={styles.disclaimer}>
+          <div className={styles.textBox}>
+            <p>
+              Just Progress helps identify and prioritize communities across the
+              United States and U.S. territories
+              that have been historically overburdened and underserved. These
+              communities will receive 40% of
+              the benefits from investments in key areas outlined by the
+              &nbsp;
+              <a
+                href={'https://www.whitehouse.gov/briefing-room/' +
+                'presidential-actions/2021/01/27/' +
+                'executive-order-on-tackling-the-climate-' +
+                'crisis-at-home-and-abroad/'}
+                target={'_blank'}
+                rel={'noreferrer'}>
+                Executive Order on Tackling the Climate Crisis at Home and
+                Abroad
+              </a>.
+            </p>
+            <p>
+              Download the Just Progress packet or explore the map below to see
+              the list of prioritized communites. To
+              learn more about how these communities were prioritized check out
+              the
+              &nbsp;
+              <a
+                href={'./methodology'}>
+                Methodology
+              </a>
+              &nbsp;
+              page.
+            </p>
           </div>
-        </section>
+          <DownloadPacket/>
+        </div>
+      </section>
+
+      <section>
+        <h2>Explore the Tool</h2>
         <MapWrapper location={location} />
-        <HowYouCanHelp />
-      </main>
+        <HowYouCanHelp/>
```

Comment:
how does this work?

Reply:
@NatHillardUSDS We should do a code in zoom walkthrough!